lib/recipe.rb

```diff
@@ -143,20 +179,22 @@
 esac
 INIT_SCRIPT
 
-file init_script_path do
-  content(init_script)
-  mode 0755
-  owner 'root'
-end
+  file init_script_path do
```

Comment:
We probably should guard the init script and service with `not_if { platform?('windows') }`